sparse/etc/bcm43xx.conf

```diff
@@ -2,10 +2,9 @@
 uart_dev = /dev/ttyHSU0
 type_id = bcm43xx
 baud_rate = 3000000
-flow = 1
-tosleep = 1
-up_hci = 1
-reg_hci = 1
-no_flush = 1
-use_lpm = 1
+tosleep = 10
+reg_hci = true
+lpm = false
+fork = true
 bdaddr = /data/misc/bluetooth/bluetooth_bdaddr
+fw_patch = /etc/firmware/bt/BCM4356A2_001.003.015.0084.0268_ORC.hcd
```

Comment:
is this correct?

Reply:
Ah, i have a symlink for /etc/firmware...